src/summary/history/allergy-card-level-three.component.tsx

```diff
@@ -0,0 +1,225 @@
+import React from "react";
+import { match } from "react-router";
+import SummaryCard from "../cards/summary-card.component";
+import style from "./allergy-card-level-three.css";
+import {
+  getAllergyAllergenByConceptUuid,
+  getAllergyReaction
+} from "./allergy-intolerance.resource";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+
+const DRUGALLERGENCONCEPT: string = "162552AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
```

Comment:
I think @joeldenning had recommended against using the all-caps for constants approach though I can't remember his reasoning. @fatmali , do you have any recommendations on this?

Reply:
it's a convention to have caps for constants in many programming languages. However, the words should be separated by an underscore